lib/chef-dk/policyfile_services/export_repo.rb

```diff
@@ -242,14 +242,14 @@ def create_client_rb
             use_policyfile true
             policy_document_native_api true
 
-            # In order to use this repo, you need a version of Chef Client and Chef Zero
+            # In order to use this repo, you need a version of Chef Infra Client and Chef Zero
```

Comment:
I notice the Dist constants are not being used here. Is `require "chef-dk/dist"` something we do not want to use in the `policyfile_services` classes?

Reply:
So I sorta noped away from this file after looking at it. It seemed to me that when a community distro exported repos and cookbooks it should still say "hey I'm built for chef". Maybe that's just me, but for now I didn't want to touch this. It's probably a larger conversation for the business as to what an exported artifact should say it' built for.

Reply:
🤔 Could be, yea. I imagine it's fine to come back to this later if we figure out better words.